README.md

```diff
@@ -1 +1,25 @@
-# node-process-lab
\ No newline at end of file
+# Access Global and Process Variables Lab
+
+In this lab, you'll need to demonstrate how to access `global` and `process` variables.
```

Comment:
can we have this intro frame this lab with some sort of real world tie instead of just listing the objects they'll need to work with and the difference between them? its much more motivating for students to feel like theyre doing something they'll use in the real world. if we can frame this like they're a systems engineer or something that will make a student feel like theyre doing more than just practicing but solving a real world problem